app/Models/Post.php

```diff
@@ -336,21 +336,12 @@ public function siblings()
     }
 
     /**
-     * Transform the post model for Blink.
+     * Transform this post for Customer.io.
      *
      * @return array
      */
     public function toCustomerIoPayload()
     {
-        // Blink expects quantity to be a number.
-        $quantity = $this->quantity === null ? 0 : $this->quantity;
-
-        // Bypass Campaign->cause accessor method so the value isn't converted to an array
-        // which Customer.io does not support.
-        $campaign_cause = optional($this->signup->campaign)->getAttributes()[
-            'cause'
-        ];
```

Comment:
I decided to replace this with an `implode(',', $campaign->cause)` call below because I think it makes the intention clearer (that we want this as a CSV), rather than just relying on that implicitly being the format stored in the database. It's also, conveniently, a little bit more concise!